packages/cli/internal/pkg/aws/cwl/get_logs.go

```diff
@@ -51,9 +51,22 @@ func (c Client) GetLogsPaginated(input GetLogsInput) LogPaginator {
 }
 
 func formatEvents(events []types.FilteredLogEvent) []string {
-	logs := make([]string, len(events))
-	for i, event := range events {
-		logs[i] = formatEvent(event)
+	logsByStream := make(map[string][]*types.FilteredLogEvent)
+	for index, _ := range events {
+		event := events[index]
```

Comment:
If we take the actual event then the pointer will only refer to the very last event